src/models/Projection.ts

```diff
@@ -140,6 +161,13 @@ export class Projection {
     return this.cumulativeDeaths[this.cumulativeDeaths.length - 1];
   }
 
+  get nonCovidICUCapacity() {
+    return (
+      this.totalICUCapacity! *
```

Comment:
I don't like `!`s...  Seems like either we should:

```
import { assert} from 'utils';
...
assert(this.totalICUCapacity !== null, 'Need totalICUCapacity');
```

or else make this property nullable and properly handle the null case.

Reply:
yeah it is nullable, i guess i'm not sure what the ramifications are like for nullable